src/runtime/lpython_builtin.py

```diff
@@ -638,6 +638,51 @@ def _lpython_str_lower(x: str) -> str:
     return res
 
 @overload
+def _lpython_str_find(s: str, sub: str) -> i32:
+    s_len :i32
+    sub_len :i32
+    s_len = len(s)
+    sub_len = len(sub)
+    flag: bool
+    flag = False
+    res: i32
+    res = -1
+    lps: list[i32]
+    i :i32
+    len: i32
```

Comment:
```suggestion
    s_len :i32; sub_len :i32; flag: bool;
    res: i32; i: i32; len: i32;
    lps: list[i32]

    s_len = len(s)
    sub_len = len(sub)
    flag = False
    res = -1
```

Reply:
I would recommend to name your variable as `len_` i.e., something different from intrinsic `len`.

Reply:
now everything is ready now.
but, i don't know what happend with windows built!